invalid json and incorrect api version

GoEddie opened the pull request:
The latest version of the vm extensions api is 2017-03-30, the one used in this example is not valid

azurecla commented:
Hi __@stuartleeks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stuartle). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@stuartleeks : Thanks for your contribution to the Azure documentation! The author, @NarayanAnnamalai, has been notified to review your proposed change.

jimdial commented:
@stuartleeks : Thanks for the edit. #sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

tysonn commented:
The listed author of this is   authors="NarayanAnnamalai". Jim, please contact azdocprs to request this be merged. Only the listed author can use the sign off comment to get this merged.

NarayanAnnamalai commented:
#sign-off

GoEddie committed:
invalid json and incorrect api version

The latest version of the vm extensions api is 2017-03-30, the one used in this example is not valid
